dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/statelisteners/ElasticListener.java

```diff
@@ -445,6 +445,7 @@ private static void copyEntryProperties(Entry detachedEntry, Entry entry) {
         // This is some weird hack to always set the topic (which is either automatic or manual) into the ES topicAutomatic property for search table
         // This is to avoid indexing both topicAutomatic and topicManual and having the frontend choose which one to display
         detachedEntry.setTopicAutomatic(entry.getTopic());
+        detachedEntry.setTopicSelection(entry.getTopicSelection());
```

Comment:
May need to update the comment above, since we have automatic, ai, and manual now.